Atomic/install.py

```diff
@@ -78,9 +85,16 @@ def __init__(self):
         super(Install, self).__init__()
 
     def install(self):
-        debug = self.args.debug
         if self.args.debug:
             util.write_out(str(self.args))
+        storage_set = False if self.args.storage is None else True
+        storage = _storage if not storage_set else self.args.storage
+        if storage == 'ostree':
+            self.args.system = True
```

Comment:
I don't think just because you are using ostree, we should set self.args.system.  You could be able to install an ostree container without running through the system stuff.